File: fizzbuzz.js
var fizzBuzz = function (n) {
  if ( n % 15 === 0 ) {
    console.log("fizzbuzz");
  } else if ( n % 3 === 0 ) {
    console.log("fizz");
  } else if ( n % 5 === 0 ) {
    console.log("buzz");
  }

};

fizzBuzz(15);
fizzBuzz(3);
fizzBuzz(5);

File: fizzbuzz.rb
def fizz_buzz(n)
  if n % 15 == 0
    puts "fizzbuzz"
  elsif n % 3 == 0
    puts "fizz"
  elsif n % 5 == 0
    puts "buzz"
  else
    puts n
  end
end

100.times do |n|
  fizz_buzz(n + 1)
end
